tests/test_ir.py

```diff
@@ -408,3 +409,19 @@ def test_intervals_subtract():
     assert ix2.subtract(ix) == ix
     assert ix.subtract(ix2) == Interval(x, -2, 2)
     assert ix3.subtract(ix) == ix2
+
+
+@skipif_yask
+def test_iet_conditional(fc):
+    then_body = Expression(Eq(fc[x, y], fc[x, y] + 1))
+    else_body = Expression(Eq(fc[x, y], fc[x, y] + 2))
+    conditional = Conditional(x < 3, then_body, else_body)
```

Comment:
This is nice. Is this currently only at lower level, or can it be generalized as a "stencil" at operator creation?